packages/instantsearch.js/src/widgets/hits-per-page/__tests__/hits-per-page.test.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * @jest-environment jsdom
+ */
+/** @jsx h */
+import { createSearchClient } from '@instantsearch/mocks';
+
+import instantsearch from '../../../index.es';
+import hitsPerPage from '../hits-per-page';
+
+describe('hitsPerPage', () => {
+  describe('options', () => {
+    test('throws without a `container`', () => {
```

Comment:
css classes test is gone

Reply:
(probably could be made common with a mapping from one of the class names to the other two , like done in the other existing "options" test)

Reply:
In what other options tests do you see the mapping? There's only breadcrumb for now and I don't see it.

For now I added it in the flavor-specific suite (5d6cbb565).